src/hooks/useOrgHooks.js

```diff
@@ -104,13 +102,14 @@ export function useAppData(organization) {
     }
   }, [appName, organization])
 
+  const config = useConfigSubscription(appData.convictionVoting)
   const proposals = useProposalsSubscription(appData.convictionVoting)
 
   // Stakes done across all proposals on this app
   // Includes old and current stakes
   const stakesHistory = useStakesHistorySubscription(appData.convictionVoting)
 
-  return { ...appData, proposals, stakesHistory }
+  return { ...appData, ...config, proposals, stakesHistory }
```

Comment:
Wondering if makes sense to have all config values inside the `config` object instead of spreading them. That way it may be easier to know whether the config has loaded or not.

Reply:
do you mind if we do this change after the release? don't want to touch a bunch of files right now? created https://github.com/1Hive/honey-pot/issues/59